app/views/account_reset/recovery_options/show.html.erb

```diff
@@ -10,8 +10,9 @@
     <%= c.with_item(heading: t('account_reset.recovery_options.use_device')) do %>
       <p><%= t('account_reset.recovery_options.try_another_device') %></p>
     <% end %>
-    <%= c.with_item(heading: t('account_reset.recovery_options.check_webauthn_platform')) do %>
-      <p><%= t('account_reset.recovery_options.check_webauthn_platform_info', app_name: APP_NAME) %></p>
+    <%= c.with_item(heading: t('account_reset.recovery_options.check_saved_credential')) do %>
+      <p><%= t('account_reset.recovery_options.check_webauthn_platform_info') %></p>
+      <p><%= t('account_reset.recovery_options.use_same_device') %></p>
```

Comment:
I don't really have a strong preference one way or the other, but another way we've done multiple paragraphs of text is by using an array in the YAML file and looping over it in the template.

https://github.com/18F/identity-idp/blob/809a5a696fd2009782b4c0fd4800a0acd9f1bb84/app/views/account_reset/request/show.html.erb#L5-L7

The main upside would be to keep the related content together in the YAML files.

Reply:
huh interesting will take a look